### ISSUE TITLE
Regarding Top-up Training to improve the accuracy of existing deepfake model

### ISSUE THREAD

**Issue by soumyajee** [polimi-ispl/icpr2020dfdc#78]
Thanks for the work regarding deepfake detection. Unfortunately your pretrained model as well as our self trained models are not generalizing on new social media deepfakes, and are having increased number of False Positives and False Negatives. how can we further fine tune or top-up train, the existing best models  on new viral deepfakes to improve detection rate. Also, when we are doing top-up training ourselves, the model weights are changing in such a way that for every video it starts predicting as Fake. How can we polish existing best models further ? Thanks

**Comment by CrohnEngineer** [comment]
Hey @soumyajee ,

Sorry for the late reply!
Your question, unfortunately, does not have an easy answer.  
We did some research ourselves some time ago (you can find the paper [here](https://arxiv.org/pdf/2011.07792)), discovering that some training strategies like the triplet loss and the use of heavy data augmentation pipelines can help a bit (especially in data scarcity conditions).  
However, I must say that the generalization of the video deepfake detection task is still an ongoing research topic.  
The research community is still looking for a definite answer, and so far, different research groups have come to different answers. I'll put some references here, but you better have a look yourself and find the papers you believe are more promising:
1. https://www.mdpi.com/2313-433X/9/5/89
2. https://arxiv.org/abs/2408.17065
3. https://openaccess.thecvf.com/content/ICCV2021/html/Cozzolino_ID-Reveal_Identity-Aware_DeepFake_Video_Detection_ICCV_2021_paper.html. This is an interesting take: rather than learning a model that can distinguish general video deepfakes, you can train a model tailored to the characteristics of a specific Person of Interest (POI). In this scenario, you don't need deepfake videos for training your model, and in this way, your technique becomes more robust to the specific deepfake video technique used or other post-processing that can be used to conceal forgery traces (e.g., social media filters, compression, etc.).

Best of luck with your research!
